frontend/src/pages/destructionlist/review/DestructionListReview.tsx

```diff
@@ -316,6 +339,28 @@ export function DestructionListReviewPage() {
     );
   }
 
+  /**
+   * Gets called when the co-reviewer completes reviewing the destruction list.
+   */
+  function handleCompleteCoReview() {
+    prompt(
+      `Medebeoordeling afronden`,
+      `U staat op het punt om de medebeoordeling voor vernietigingslijst ${destructionList.name} af te ronden, wilt u doorgaan?`,
+      "Opmerking",
+      "Medebeoordeling afronden",
+      "Annuleren",
+      (comment) => {
+        submitAction({
+          type: "COMPLETE_CO_REVIEW",
+          payload: {
+            comment,
```

Comment:
Is this not `list_feedback` in the backend? :thinking:

Reply:
It is, but the action maps it to the correct field name, the "payload" here is an internal type for the dispatched action.